symplyphysics/laws/thermodynamics/work_done_by_gas_at_constant_pressure.py

```diff
@@ -0,0 +1,47 @@
+from sympy import Eq, solve
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output)
+
+# Description
+# When energy is added to gas molecules and increases their kinetic energy,
+# the gas expands and does work on its surroundings. The work done by the gas
+# with constant pressure can be found by: A = p * ΔV, where
+# ΔV is the change in the volume of the gas,
+# p is pressure,
+# A is work done.
+# When the volume of a gas changes from to V1 to V2, the change
+# in the volume of the gas is ΔV = V2 - V1
```

Comment:
We usually have a line with law in the description, like this:
`Law: A = p * (V2 - V1)`
Please add it to conform to other laws descriptions.